### PR TITLE
Fixing base URL for R2

### PR DESCRIPTION
**Pull request by CsatariGergely**
Fixing the 404 issue of the breadcrumb navigation in R2.

/assign @efiacor @liamfallon 

This supposed to fix the issue for the R2 branch pointed out by @efiacor in https://github.com/nephio-project/docs/pull/128#pullrequestreview-2101516416

As an extra I'm adding the link fixes to R2 merged to main in https://github.com/nephio-project/docs/pull/130 (https://github.com/nephio-project/docs/pull/135/commits/69c79e298d25b4a50677099eee575986071b1f2a)

Link checks for the [LFN Wiki](https://wiki.lfnetworking.org/) are disabled as the wiki is down. [LF IT ticket](https://jira.linuxfoundation.org/plugins/servlet/desk/portal/2/IT-26874) was created.

### CODE REVIEW

**Comment by liamfallon** [comment]
I downloaded this PR to test it but when I ran `hugo serve`, the result was:

```
>  hugo serve

Hugo provides its own webserver which builds and serves the site.
While hugo server is high performance, it is a webserver with limited options.

...skipped lines....

Use "hugo server [command] --help" for more information about a command.

Error: command error: Unable to locate config file or config directory. Perhaps you need to create a new site.
Run `hugo help new` for details.
```

File: config.toml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-baseURL = 'https://nephio.org/'
+baseURL = 'https://r2.docs.nephio.org/'
```
Comment (efiacor) [inline config.toml:1]: I guess these will tie in to the drop down menu when it gets merged right?

**Comment by liamfallon** [comment]
> I downloaded this PR to test it but when I ran `hugo serve`, the result was:
> 
> ```
> >  hugo serve
> 
> Hugo provides its own webserver which builds and serves the site.
> While hugo server is high performance, it is a webserver with limited options.
> 
> ...skipped lines....
> 
> Use "hugo server [command] --help" for more information about a command.
> 
> Error: command error: Unable to locate config file or config directory. Perhaps you need to create a new site.
> Run `hugo help new` for details.
> ```

Never mind, my mistake, I pulled the wrong PR.

File: config.toml (hunk 1 above)
Comment (CsatariGergely) [reply config.toml:1]: The drop downs are added in https://github.com/nephio-project/docs/pull/128 and in https://github.com/nephio-project/docs/pull/133

**Review by liamfallon — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**